src/librbd/internal.cc

```diff
@@ -3445,6 +3445,23 @@ int invoke_async_request(ImageCtx *ictx, const std::string& request_type,
     return r;
   }
 
+  int flush_cache(ImageCtx *ictx)
```

Comment:
Is there any real difference between this and the existing ```flush(ImageCtx *)``` method?  It even already has an existing perf counter.